config/buildbuddy.integration.yaml

```diff
@@ -0,0 +1,7 @@
+# NOTE: most integration test config is set via flags.
```

Comment:
can we put these configs in a test directory instead so they are clearly not-for-use? Maybe test/config?

Reply:
Deleted this file since we're re-parsing config flags now